packages/manager/apps/pci-ai-tools/src/components/order/app-scaling/AutoScalingForm/AutoScalingForm.component.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { useCallback, useMemo } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useForm, useWatch } from 'react-hook-form';
+import { zodResolver } from '@hookform/resolvers/zod';
+import { Form } from '@datatr-ux/uxlib';
+
+import Price from '@/components/price/Price.component';
+import { AppPricing, Scaling } from '@/types/orderFunnel';
+import {
+  createScalingSchema,
+  getInitialValues,
+  toScaling,
+  ScalingFormValues,
+} from '@/lib/scalingHelper';
```

Comment:
Should be next to the form:
- /AutoScalingForm/autoScalingForm.schema.ts
- /AutoScalingForm/autoScalingFormHelper.ts